src/loader-components/_ExampleTemplate.jsx

```diff
@@ -4,6 +4,7 @@ import classNames from 'classnames/bind';
 import { ThemeContext } from 'terra-application/lib/theme';
 import IconChevronLeft from 'terra-icon/lib/icon/IconChevronLeft';
 import IconChevronRight from 'terra-icon/lib/icon/IconChevronRight';
+import * as KeyCode from 'keycode-js';
```

Comment:
let's only import the codes we need

Reply:
Updated here: here: [d830ef0](https://github.com/cerner/terra-dev-site/pull/281/commits/d830ef07a820fb004bc59f65cde8f20ee7052353#).